client/my-sites/upgrades/cart/cart-item.jsx

```diff
@@ -51,6 +52,27 @@ module.exports = React.createClass( {
 		} );
 	},
 
+	monthlyPrice: function() {
+		const { cost, product_slug: productSlug, currency } = this.props.cartItem,
+			// A hacky way to check if we are in signup, since for some reason cartItem.extra.context
+			// is missing in that case (it seems cartItem.extra is overwritten somewhere between)
+			isInSignup = this.props.cartItem.extra && this.props.cartItem.extra.context !== 'calypstore',
```

Comment:
`cartItem.extra.context` should be set to `signup` according to [`cart.js`](https://github.com/Automattic/wp-calypso/blob/master/client/lib/signup/cart.js#L28), but for some reason it's not when the cart item is displayed during the checkout. There's only `last_server_response_date`, so it looks like `extra` is overwritten somewhere? I couldn't find where though :/